shared/ff_util/include/ff_util/ff_component.h

```diff
@@ -89,7 +89,7 @@ class FreeFlyerComponent {
 
   void AssertFault(FaultKeys enum_key,
                    std::string const& message,
-                   ros::Time time_fault_occurred = ROS_TIME_NOW());
+                   ros::Time time_fault_occurred = rclcpp::Clock().now());
```

Comment:
FF_TIME_NOW?

Reply:
Can't use this here because it needs to be static, which FF_TIME_NOW() isn't :/

Reply:
https://github.com/nasa/astrobee/issues/637